elements/lisk-validator/src/formats.ts

```diff
@@ -42,7 +44,8 @@ export const additionPublicKey = (data: string): boolean => {
 
 export const amount = isNumberString;
 
-export const csv = isCsv;
+export const csv = (data: string) =>
+	isCsv(data) && data.split(',').length <= maxCSVValues;
```

Comment:
I think adding 1000 limit to the format itself doesn't make much sense.
Prefer to keep the current format or remove `csv` since it is not used

Reply:
I tried to migrate logic of existing formats to `lisk-validator`, so we can later use it within the framework. If you prefer we can remove it.